core/tests/widgets/test_container_debug_mode.py

```diff
@@ -0,0 +1,76 @@
+from pytest import MonkeyPatch
+from travertino.colors import color
+
+import toga
+
+
+# test that a container-like widget in normal mode has a default background
+def test_box_no_background_in_normal_mode():
+    """A Box has no default background."""
+    # Disable layout debug mode
+    with MonkeyPatch.context() as mp:
+        mp.setenv("TOGA_DEBUG_LAYOUT", "0")
+        box = toga.Box()
+        # assert that the bg is default
+        assert hasattr(box.style, "background_color")
+        assert not box.style.background_color
```

Comment:
A check of an actual value, rather than "falseness" would be preferable here - that avoids any odd situations where something like `rgb(0,0,0)` evaluates as "false" (I don't think it does, but just in case)